app/design/adminhtml/Magento/backend/web/js/theme.js

```diff
@@ -312,8 +312,9 @@ define('globalNavigation', [
 
 define('globalSearch', [
     'jquery',
+    'Magento_Ui/js/lib/key-codes',
     'jquery/ui'
```

Comment:
Could you replace using `jquery/ui` to what component really needed?
```suggestion
    'jquery-ui-modules/widget'
```

I don't see any other other usages here rather than jquery widget, and we'll fix the following warning for this specific case:
https://github.com/magento/magento2/blob/9544fb243d5848a497d4ea7b88e08609376ac39e/lib/web/jquery/compat.js#L43-L48